developer_manual/app_development/bootstrap.rst

```diff
@@ -52,19 +52,24 @@ The class **must** extend ``OCP\AppFramework\App`` and may optionally implement
     namespace OCA\MyApp\AppInfo;
 
     use OCA\MyApp\Listeners\UserDeletedListener;
-    use OCA\MyApp\Notifications\Notifier;
+    use OCA\MyApp\Notification\Notifier;
     use OCP\AppFramework\App;
+    use OCP\AppFramework\Bootstrap\IBootContext;
     use OCP\AppFramework\Bootstrap\IBootstrap;
-    use OCP\Notification\IManager;
+    use OCP\AppFramework\Bootstrap\IRegistrationContext;
     use OCP\User\Events;
 
     class Application extends App implements IBootstrap {
 
+        public function __construct() {
+            parent::__construct('myapp');
+        }
+
         public function register(IRegistrationContext $context): void {
             // ... registration logic goes here ...
 
             // Register the composer autoloader for packages shipped by this app, if applicable
-            include_once __DIR__ . '/../../vendor/autoload.php'
+            @include_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Errors should not be hidden here via `@`. The `;` is still correct. 👍

Reply:
This is from https://github.com/nextcloud/documentation/blob/master/developer_manual/app_development/bootstrap.rst#L141, old ``app.php`` example.
I would also change ``app.php`` example since NC19 is still supported.